src/psyclone/tests/dynamo0p3_builtins_test.py

```diff
@@ -281,13 +281,15 @@ def test_builtin_integer_field_arg():
     # Define the built-in name and test file
     test_builtin_name = "X_minus_bY"
     test_builtin_file = "15.2.4_" + test_builtin_name + "_builtin.f90"
-    dynamo0p3_builtins.BUILTIN_DEFINITIONS_FILE = \
-        os.path.join(BASE_PATH, "invalid_builtins_mod.f90")
+    builtins_file = os.path.join(BASE_PATH, "invalid_builtins_mod.f90")
+    monkeypatch.setattr(dynamo0p3_builtins, "BUILTIN_DEFINITIONS_FILE",
+                        value=builtins_file)
     _, invoke_info = parse(
         os.path.join(BASE_PATH,
                      test_builtin_file),
         api=API)
-    dynamo0p3_builtins.BUILTIN_DEFINITIONS_FILE = old_name
+    monkeypatch.setattr(dynamo0p3_builtins, "BUILTIN_DEFINITIONS_FILE",
```

Comment:
I don't think you need this bit any more or does the call to PSyFactory require the valid definitions file?

Reply:
You're right, I don't. Removed.